ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -253,12 +254,163 @@ hideall "--sync--"
+10219.2 "Reversal Of Forces" sync /:Queen's Warrior:5829:/
+10219.2 "--sync--" sync /:Queen's Warrior:5A94:/
+10228.4 "Optimal Offensive" sync /:Queen's Knight:581[9A]:/
+10230.3 "Above Board" sync /:Queen's Warrior:5826:/
+10231.0 "Unlucky Lot" #sync /:Aetherial Sphere:581D:/
+10233.4 "Lots Cast" #sync /:Aetherial Bolt:5828:/
+10236.4 "Lots Cast" #sync /:Aetherial Burst:5827:/
+10237.6 "Lots Cast" #sync /:Aetherial Burst:5BB7:/
+
+10246.5 "Boost" sync /:Queen's Warrior:582D:/
+10254.7 "Blood And Bone" sync /:Queen's Warrior:5831:/
+10269.9 "Relentless Battery" sync /:Queen's Warrior:5825:/
+10277.0 "Reversal Of Forces" sync /:Queen's Warrior:5829:/
+10285.1 "Shield Omen/Sword Omen" sync /:Queen's Knight:581[45]:/
+10286.2 "Winds Of Weight" sync /:Queen's Warrior:582A:/
+10286.2 "Winds Of Fate" sync /:Queen's Warrior:582C:/
+10286.2 "Weight Of Fortune" sync /:Queen's Warrior:582B:/
+10293.3 "Optimal Play" sync /:Queen's Knight:581[67]:/
+10295.3 "Boost" sync /:Queen's Warrior:582D:/
+10303.5 "Blood And Bone" sync /:Queen's Warrior:5831:/
+10312.6 "Rapid Sever" sync /:Queen's Warrior:5832:/
+
+# loop, unrolled so there can be an explicit enrage
+10334.8 "Relentless Battery" sync /:Queen's Warrior:5825:/
+10341.0 "Shield Omen/Sword Omen" sync /:Queen's Knight:581[45]:/
+
+10341.9 "Bombslinger" sync /:Queen's Warrior:5AFE:/
+10349.0 "Reversal Of Forces" sync /:Queen's Warrior:5829:/
+10358.3 "Optimal Offensive" sync /:Queen's Knight:581[9A]:/
+10359.9 "Unlucky Lot" #sync /:Aetherial Sphere:581D:/
+10360.1 "Above Board" sync /:Queen's Warrior:5826:/
+10360.1 "--sync--" sync /:Queen's Warrior:5B72:/
+10363.1 "Lots Cast" #sync /:Aetherial Burst:5B6D:/
+10366.1 "Lots Cast" #sync /:Aetherial Bolt:5B6C:/
+10367.3 "Lots Cast" #sync /:Aetherial Bolt:5BB7:/
+
+10376.2 "Boost" sync /:Queen's Warrior:582D:/
+10384.4 "Blood And Bone" sync /:Queen's Warrior:5831:/
+10399.6 "Relentless Battery" sync /:Queen's Warrior:5825:/
+10406.7 "Reversal Of Forces" sync /:Queen's Warrior:5829:/
+10414.8 "Shield Omen/Sword Omen" sync /:Queen's Knight:581[45]:/
+10415.9 "Winds Of Weight" sync /:Queen's Warrior:582A:/
+10415.9 "Winds Of Fate" sync /:Queen's Warrior:582C:/
+10415.9 "Winds Of Fate" sync /:Queen's Warrior:582C:/
+10415.9 "Weight Of Fortune" sync /:Queen's Warrior:582B:/
+10423.0 "Optimal Play" sync /:Queen's Knight:581[67]:/
+10425.0 "Boost" sync /:Queen's Warrior:582D:/
+10433.2 "Blood And Bone" sync /:Queen's Warrior:5831:/
+10442.3 "Rapid Sever" sync /:Queen's Warrior:5832:/
+
+# If either die, they will start casting enrage.
+# ??? this timing is a guess for the natural push.
+10451.1 "--sync--" sync /:5823:Queen's Knight/ window 300,10
+10451.1 "--sync--" sync /:5833:Queen's Warrior/ window 300,10
+10461.1 "Blood And Bone Enrage" sync /:Queen's Knight:5823:/
+# these ability ids are unknown, and timing is a guess from soldier/gunner enrage
+10464.2 "Blood And Bone" #sync /:Queen's Soldier:???:/ window 10,10 # x infinity
```

Comment:
5833 and 5823 are the IDs that starts the enrage.
The damage is from Spell IDs 5B9E and 5B9D.

313.5 "Blood and Bone" sync /:Queen's Warrior:5833:/
313.5 "Blood and Bone" sync /:Queen's Knight:5823:/
316.6 "Blood and Bone" sync /:Queen's Warrior:5B9E:/
316.6 "Blood and Bone" sync /:Queen's Knight:5B9D:/
319.7 "Blood and Bone" sync /:Queen's Warrior:5B9E:/
319.7 "Blood and Bone" sync /:Queen's Knight:5B9D:/


Here is an encounter where these goes off: https://www.fflogs.com/reports/LZPRYhkt3NBXW4aF#fight=7&type=casts&view=events&hostility=1&eventstart=8518857

Reply:
Thank you!! How are you finding these logs so quickly? I have such a hard time digging up unknown zones in fflogs.

Reply:
FFlogs has a zoneid for Delubrum Reginae (Normal + Savage) of 39, you can query for 24 pages of those at a time.

Fortunately I did this early on before you had this up and was able to grab logs from beyond what is today's 24 page limit as at the time logs from back on Feb 8 were only on page 11 or so.

The Styg Warrior as the hardest to find, but I found a trick to it. Compare Dahu and Trinity Seeker encounters in a single log and you find 1 player missing, then look that player up on FFlogs (hoping they are not anonymous and that they uploaded the Warrior encounter), then browse through their logs to see if they are uploading personal logs (a lot of their logs may have been logged by someone else or the guild they are in), one I find their fflog's account I can browse all of their public logs and get logs back as late as when ACT was in beta for the patch.